data/apply-planet_osm_line.sql

```diff
@@ -33,6 +33,8 @@ CREATE INDEX planet_osm_line_transit_geom_15_index ON planet_osm_line USING gist
 
 CREATE INDEX planet_osm_line_waterway_geom_index ON planet_osm_line USING gist(way) WHERE waterway IN ('canal', 'river', 'stream', 'dam', 'ditch', 'drain');
 
+CREATE INDEX planet_osm_line_piste_geom_index ON planet_osm_line USING gist(way) WHERE tags ? 'piste:type';
```

Comment:
Should we make the index here a partial on the geometry that matches the query filter exactly?